cosmovisor/cmd/cosmovisor/version.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/cosmovisor"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+	tmcli "github.com/tendermint/tendermint/libs/cli"
+)
+
+func init() {
+	versionCmd.Flags().StringP(OutputFlag, "o", "text", "Output format (text|json)")
+	rootCmd.AddCommand(versionCmd)
+}
+
+var (
+	// Version represents Cosmovisor version value. Overwritten during build
+	Version = "1.1.0"
+	// OutputFlag defines the output format flag
+	OutputFlag = tmcli.OutputFlag
+)
+
+var versionCmd = &cobra.Command{
+	Use:          "version",
+	Short:        "Prints the version of Cosmovisor.",
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		logger := cmd.Context().Value(cosmovisor.LoggerKey).(*zerolog.Logger)
+
+		if val, err := cmd.Flags().GetString(OutputFlag); val == "json" && err == nil {
+			return printVersionJSON(logger, args)
+		}
+
+		return printVersion(logger, args)
+	},
+}
+
+func printVersion(logger *zerolog.Logger, args []string) error {
```

Comment:
should we use the logger interface from tendermint to keep consistent with sdk?

Reply:
The problem is that interface does not allow us to disable the logger.

Reply:
you would pass it a noop logger, this is what we use to do in tendermint

Reply:
Correct -- zerolog has a no-op logger.